gensim/models/keyedvectors.py

```diff
@@ -1063,6 +1067,134 @@ def get_keras_embedding(self, train_embeddings=False):
         )
         return layer
 
+    def simple_average(self, sent):
+        """
+        Mixture of word-vectors to make sentence-vector.
+
+        Attains this by simple averaging of all word-vectors.
+
+        Examples
+        --------
+        >>> sent = iter([['sample', 'sentence', 'cat', 'dog'], ['time', 'computers', 'expensive']])
+        >>> trained_model.simple_average(sent)
+
+
+        Parameters
+        ------------
+        sent: iterator of list of tokens of sentences
+            Corpus of sentences.
+
+        Returns
+        --------
+        sents_emd: class: `gensim.models.keyedvectors.Word2VecKeyedVectors`
+            Sentence embeddings of each sentence in the same order as given.
+
+        """
+        sents_emd = []
```

Comment:
you should make embedding **for one** `sentence` (not for sequence of sentences), or support both variant